swim/memberlist.go

```diff
@@ -228,6 +243,16 @@ func (m *memberlist) MakeChange(address string, incarnation int64, status string
 		Status:            status,
 		Timestamp:         util.Timestamp(time.Now()),
 	}})
+
+	if len(changes) > 0 {
+		m.logger.WithFields(bark.Fields{
+			"update": changes[0],
+		}).Debugf("ringpop member declares other member %s", changes[0].Status)
+
+		m.node.emit(ChangeMadeEvent{changes[0]})
```

Comment:
Nope, good question. I initially added the event so I could add all the logs to an event handler. Now this event is not even used so I am inclined to remove this event completely from this PR.